README.md

```diff
@@ -121,39 +121,20 @@ The following tables lists the configurable parameters of the Orchestrate chart
 | `api.environment`                                        | Environment variables passed to Orchestrate API containers                                                             | `{}`                |
 | `api.environmentSecrets`                                 | Environment variables (as Kubernetes secrets) passed to Orchestrate API containers                                     | `{}`                |
 | `api.existingSecret`                                     | If specified, extra environment variables will be added from externally created secret                                 | `nil`               |
-| `api.initMigrate.enabled`                                | Run migration initialization job                                                                                       | `true`              |
-| `api.initMigrate.backoffLimit`                           | Number of retries before considering the initialization migration initialization as failed                             | `6`                 |
-| `api.migrate`                                            | Run migration script                                                                                                   | `true`              |
+| `migrate.enabled`                                | Run migration job                                                                                       | `true`              |
```

Comment:
I think migration should not be optional at any time so that I would rather remove it